shed/utils.py

```diff
@@ -24,6 +26,9 @@ def to_event_model(data, output_info):
     This is only for demonstration/example use, do not use for production.
     """
     es = EventStream(md={'source': 'to_event_model'}, output_info=output_info)
+    # add some metadata
+    es.md = md
```

Comment:
You may want to just run a `md.update({'source': 'to_event_model'}) before the init of `EventStream`. I don't know about it now, but we may do things with the init where we will want the md at init time to be accurate.

Reply:
oops yes, sounds good. what you have looks better in #75!

Reply:
Cool, I'll merge once it's updated.